conda_environment.yml

```diff
@@ -10,6 +10,11 @@ dependencies:
   - sphinx
   - pip
   - cmaps
+  - geocat-datafiles
+  - geocat-comp
```

Comment:
geocat-datafiles and geocat-comp seem like used in test.py, but they are not actually requirements for the development of geocat-viz. I am not sure if should have them here, and if not, how we should handle test.py. 
Any thoughts @anissa111 ?

Reply:
Since we are removing the testing file, I removed these from the environment!